### PR TITLE
Allows disabling of remote src markdown

### LINKED ISSUE #69: sanitize always forced to true
### Description
When binding a markdown file as the source of content, the `sanitize` flag is always set to `true`.

marked.js supports directly embedding html eg:
```
Name   | Description
-------|---------------
color  | The color. Allowable values are: <br/><ul><li>red</li><li>blue</li></ul>
```

[Line 326](https://github.com/PolymerElements/marked-element/blob/master/marked-element.html#L326) forces the sanitize property to true.

Blame shows commit [c91e398](https://github.com/PolymerElements/marked-element/commit/c91e39801e5b2b4656cd15db5209433fe06b4ae6#diff-f0b9e140535ccafe29cfe484f7fc4a11R317) as changing that line.

I'm not sure if this was intentional... mabye it existed prior to that commit . Does anyone know if there's an intent here? Can I submit a PR?

### Expected outcome

HTML inside markdown renders

### Actual outcome

HTML inside markdown is shown as text

### Live Demo

n/a

### Steps to reproduce

bind a markdown file to marked-element's source, include some html inside the md file

### Browsers Affected
- [x] Chrome
- [x] Firefox
- [x] Safari 9
- [x] Safari 8
- [x] Safari 7
- [x] Edge
- [x] IE 11
- [x] IE 10

### PR DESCRIPTION
**Pull request by e111077**
Fixes #69

Allows user to specify whether or not they want to override the default sanitization.

### CODE REVIEW

File: marked-element.html [hunk 1]
```diff
@@ -149,6 +149,17 @@
         value: false
       },
       /**
+       * If true, overrides the default sanitization of any markdown received by
+       * a request and allows fetched unsanitized markdown
+       *
+       * e.g. fetching markdown via `src` that has HTML.
+       * Note: this value overrides `sanitize` if a request is made.
+       */
+      overrideRemoteSanitization: {
```
Comment (stramel) [inline marked-element.html]: Would disableRemoteSanatization be better?

**Review by stramel — APPROVED** [review]

File: marked-element.html [hunk 2]
```diff
@@ -149,6 +149,17 @@
         value: false
       },
       /**
+       * If true, overrides the default sanitization of any markdown received by
+       * a request and allows fetched unsanitized markdown
+       *
+       * e.g. fetching markdown via `src` that has HTML.
+       * Note: this value overrides `sanitize` if a request is made.
```
Comment (stramel) [inline marked-element.html]: Doesn't actually override `sanatize`

File: marked-element.html (hunk 1 above)
Comment (e111077) [reply marked-element.html]: Yes, I was struggling with this name so hard

Comment (e111077) [reply marked-element.html]: updated name!

**Pull request merged**